sources/packages/backend/apps/api/src/testHelpers/mocked-providers/queue-module-mock.ts

```diff
@@ -10,8 +10,11 @@ import { QueueNames } from "@sims/utilities";
 export class MockedQueueModule {}
 
 function getMockedQueueProviders(): Provider[] {
+  const mockAddQueue = (): any => {
+    return "fake queue";
+  };
   return Object.values(QueueNames).map<Provider>((queue) => ({
     provide: `BullQueue_${queue}`,
-    useValue: {},
+    useValue: { add: mockAddQueue },
```

Comment:
Just wondering if only `useValue: mockAddQueue` was not enough.
To my knowledge, what is the idea of the `add` here?

Reply:
it is a left over. will remove.